ssm/quiet_sidekiq_workers.sh

```diff
@@ -51,7 +47,7 @@ do
     echo  "Quieting worker ${worker}"
 
     # Quiet the worker
-    quiet_worker "${worker}"
+    quiet_worker
```

Comment:
So this was done the old way to handle when there were multiple sidekiq processes running on an instance.  We used to run 2 sidekiq processes but switched to one ages ago.

I don't think this new way is an issue but just given this is a public repo, this is a breaking change (actually, I guess all of the changes here are breaking!)

Reply:
Good point.  I think I can add a condition based on: 
```
cat /etc/system-release
Amazon Linux release 2 (Karoo)
```
vs
```
cat /etc/system-release
Amazon Linux AMI release 2018.03
```
and look for systemctl vs initctl commands.

I will put back the pid based signaling and update the readme.  

I don't have a proper EBv2 env with multiple sidekiq pids to test this on though :(